optima/model.py

```diff
@@ -7,18 +7,19 @@ def model(simpars=None, settings=None, verbose=None, benchmark=False, die=True):
     """
     Runs Optima's epidemiological model.
     
-    Version: 2016jan31
+    Version: 2016feb02
     """
     
     
     if benchmark: starttime = tic()
     
+    
     ###############################################################################
     ## Setup
     ###############################################################################
 
     # Hard-coded parameters that hopefully don't matter too much
-    eps = 1e-3 # Define another small number to avoid divide-by-zero errors
+    eps = 1.0101e-3 # Define another small number to avoid divide-by-zero errors
```

Comment:
i kept getting these incorrect results of 1e-3 and i thought it might be an eps, but it wasn't -- anyway, this has been superseded anyway, eps is now set by settings.eps